Gruntfile.js

```diff
@@ -4,7 +4,65 @@ module.exports = function (grunt) {
 
     require('time-grunt')(grunt);
 
+    var derequire = require('derequire');
+    var envify = require('envify/custom');
+
+    var envifyTransform = envify({NODE_UNIQUE_ID: 0});
+
+    // String to be prepended to source files
+    var banner = "/*! shortId.js\n https://github.com/dylang/shortid\n Copyright (c) Dylan Greene\n License: MIT +no-false-attribs License\n*/\n";
+
     grunt.initConfig({
+        browserify: {
+            dist: {
+                src: ['lib/shortid.js'],
+                dest: 'shortId.js'
```

Comment:
I'm ok with that, but in all your samples you always write stuff like `var shortId = require('shortid')`, so I wanted to be consistent (the name of the globally exported variable is shortId).

Btw, Macs are case-sensitive as well.